src/core/Translator.test.js

```diff
@@ -0,0 +1,38 @@
+import Core from '../../src/core/index.js'
+import russian from '../../src/locales/ru_RU.js'
+import english from '../../src/locales/en_US.js'
+
```

Comment:
Locale files are probably outdates, because locale settings are per-plugin now. Not a major issue, but just to keep this in mind :)